### PR TITLE
fix checking media and delete usued media and sync issue#32

### PR DESCRIPTION
**Pull request by cecini**
https://github.com/ankicommunity/anki-sync-server/issues/32
After checking media and delete usued media ,the Window pops out,Ankiweb encountered a problem.Please try again in a few minutes

### CODE REVIEW

File: src/ankisyncd/sync_app.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def _adopt_media_changes_from_zip(self, zip_file):
         # Remove media files that were removed on the client.
         media_to_remove = []
         for normname, ordinal in meta:
-            if ordinal == '':
+            if ordinal == None  or ordinal == '':
```
Comment (mrozigor) [inline src/ankisyncd/sync_app.py:239]: Maybe just `if not ordinal:`?

Comment (cecini) [reply src/ankisyncd/sync_app.py:239]: Yes

**Comment by VikashKothary** [comment]
LGTM

**Review by VikashKothary — APPROVED** [review]
LGTM

**Review by VikashKothary — APPROVED** [review]

**Pull request merged**